### PR TITLE
Student Finance Calculator - outcome changes

### PR DESCRIPTION
**Pull request by ChrisBAshton**
This PR changes some of the outcome text for the [Student Finance Calculator](https://www.gov.uk/student-finance-calculator):

| Outcome | Live (current) URL | Draft URL |
|---------|--------------------|-----------|
| 1. UK full-time students that enter household income and aren’t eligible for the maximum maintenance loan or any other grants | [Live (current) URL](https://www.gov.uk/student-finance-calculator/y/2021-2022/uk-full-time/9250.0/away-outside-london/50000.0/no/none-of-the-above) | [Draft URL](https://smart-answer-student-lo-33u1nm.herokuapp.com/student-finance-calculator/y/2021-2022/uk-full-time/9250.0/away-outside-london/50000.0/no/none-of-the-above) |
| 2. UK full-time students that enter household income and aren’t eligible for the maximum maintenance loan, but are eligible for other grants | [Live (current) URL](https://www.gov.uk/student-finance-calculator/y/2021-2022/uk-full-time/9250.0/away-outside-london/250000.0/children-under-17,dependant-adult,has-disability,low-income/dental-medical-healthcare/yes) | [Draft URL](https://smart-answer-student-lo-33u1nm.herokuapp.com/student-finance-calculator/y/2021-2022/uk-full-time/9250.0/away-outside-london/250000.0/children-under-17,dependant-adult,has-disability,low-income/dental-medical-healthcare/yes) |
| 3. UK part-time students that enter household income and aren’t eligible for the maximum maintenance loan or any other grants | [Live (current) URL](https://www.gov.uk/student-finance-calculator/y/2021-2022/uk-part-time/6935.0/away-outside-london/250000.0/60.0/120.0/no/none-of-the-above) | [Draft URL](https://smart-answer-student-lo-33u1nm.herokuapp.com/student-finance-calculator/y/2021-2022/uk-part-time/6935.0/away-outside-london/250000.0/60.0/120.0/no/none-of-the-above) |
| 4. UK part-time students that enter household income and aren’t eligible for the maximum maintenance loan, but are eligible for other grants | [Live (current) URL](https://www.gov.uk/student-finance-calculator/y/2021-2022/uk-part-time/6935.0/away-outside-london/50000.0/60.0/120.0/has-disability,low-income/social-work) | [Draft URL](https://smart-answer-student-lo-33u1nm.herokuapp.com/student-finance-calculator/y/2021-2022/uk-part-time/6935.0/away-outside-london/50000.0/60.0/120.0/has-disability,low-income/social-work) |

See commits for details.

Trello card: https://trello.com/c/GZvq41x3/2674-student-finance-calculator-parental-contribution

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/flows/student_finance_calculator_flow/outcomes/_disclaimer.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 $!
 Your result is an estimate. Usually, you only get student finance if you’re doing your first higher education qualification - [check if you qualify](/student-finance/who-qualifies).
 $!
+
+Read the [guidance](http://student-finance/new-fulltime-students) about student finance and how to apply.
```
Comment (ChrisBAshton) [inline app/flows/student_finance_calculator_flow/outcomes/_disclaimer.erb:5]: TODO: Only applies to full time outcome.

File: app/flows/student_finance_calculator_flow/outcomes/outcome_uk_full_time_dental_medical_students.erb [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
     In each of your first 4 years, you could get:
 
-    <%= render partial: 'full_time_tuition', locals: { tuition_fee_amount: calculator.tuition_fee_amount, maintenance_loan_amount: calculator.maintenance_loan_amount, maintenance_grant_amount: calculator.maintenance_grant_amount, start_date: calculator.course_start } %>
-
+    - <%= format_money(tuition_fee_amount, pounds_only: true) %> [Tuition Fee Loan](/student-finance/new-fulltime-students) to pay for your course
```
Comment (ChrisBAshton) [inline app/flows/student_finance_calculator_flow/outcomes/outcome_uk_full_time_dental_medical_students.erb:9]: TODO: in the commit message, link to the file / commit this content is copied from.

File: app/flows/student_finance_calculator_flow/outcomes/_full_time_tuition.erb [hunk 3]
```diff
@@ -2,18 +2,20 @@
 
 You could get a <%= format_money(tuition_fee_amount, pounds_only: true) %> [Tuition Fee Loan](/student-finance/new-fulltime-students) each year to pay for your course.
 
-## Maintenance Loan
+<% if maintenance_loan_amount > 0 %>
```
Comment (ChrisBAshton) [inline app/flows/student_finance_calculator_flow/outcomes/_full_time_tuition.erb]: TODO: fill out commit message, and perhaps link to the commit/code where the logic is duplicated.

File: app/flows/student_finance_calculator_flow/outcomes/outcome_uk_all_students.erb [hunk 4]
```diff
@@ -15,15 +15,7 @@
...
-  ##Student finance
-
-  You could get per year:
-
-  - <%= format_money(calculator.tuition_fee_amount, pounds_only: true) %> [Tuition Fee Loan](/student-finance/parttime-students)
-
-  <% if calculator.maintenance_loan_amount > 0 %>
-    - <%= format_money(calculator.maintenance_loan_amount, pounds_only: true) %> [Maintenance Loan](/student-finance/parttime-students)
-  <% end %>
+  <%= render partial: 'tuition_maintenance_summary', locals: { tuition_fee_amount: calculator.tuition_fee_amount, maintenance_loan_amount: calculator.maintenance_loan_amount, maintenance_grant_amount: calculator.maintenance_grant_amount, start_date: calculator.course_start, max_loan_amount: calculator.max_loan_amount, tuition_fee_url: "/student-finance/parttime-students", maintenance_loan_url: "/student-finance/parttime-students" } %>
```
Comment (ChrisBAshton) [inline app/flows/student_finance_calculator_flow/outcomes/outcome_uk_all_students.erb:18]: TODO: another commit to split these parameters onto separate lines, now that there are so many of them. (Do this for all outcomes).

File: app/flows/student_finance_calculator_flow/outcomes/_full_time_tuition.erb [hunk 5]
```diff
@@ -1,3 +1,19 @@
...
-- <%= format_money(maintenance_loan_amount, pounds_only: true) %> [Maintenance Loan](/student-finance/new-fulltime-students) to pay living costs
+You could get a <%= format_money(tuition_fee_amount, pounds_only: true) %> [Tuition Fee Loan](/student-finance/new-fulltime-students) each year to pay for your course.
+
+## Maintenance Loan
+
+You could get a <%= format_money(maintenance_loan_amount, pounds_only: true) %> [Maintenance Loan](/student-finance/new-fulltime-students) to contribute towards your living costs.
+
+How your Maintenance Loan is calculated:
+
+&nbsp; <%= format_money(max_loan_amount, pounds_only: true) %> (the maximum Maintenance Loan available)
```
Comment (ChrisBAshton) [inline app/flows/student_finance_calculator_flow/outcomes/_full_time_tuition.erb]: TODO: add a test?

**Review by beccapearce — APPROVED** [review]

**Pull request merged**